grouper/repositories/permission_grant.py

```diff
@@ -175,6 +198,37 @@ def service_account_grants_for_permission(self, name):
             for g in grants.all()
         ]
 
+    def permission_grants_for_service_account(self, name):
+        # type: (str) -> List[ServiceAccountPermissionGrant]
+        """Return all permission grants for a service account.
+
+        TODO(rra): Currently does not expand permission aliases, and therefore doesn't match the
+        graph behavior.  Use with caution until that is fixed.
```

Comment:
Is there any correct usage of function? If not, feels ideal to actually guarantee this doesn't get used accidentally

Reply:
It's used by `grouper-ctl` to check whether a service account has the `USER_ADMIN` permission and thus can create other service accounts. The only thing it doesn't support is permission aliases, which are a graph-only feature (which means they're just in general not supported by a bunch of stuff right now).